cypress/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+export const getTestSelector = (selectorId: string) => `[data-cy=${selectorId}]`
+
+// 1. Go to featureFlags/index.tsx
+// 2. Find the FeatureFlag enum
+// 3. Copy the string that correspond with the flag you want to enable and pass it here
```

Comment:
I would just import this enum, but I can't import from a .tsx file and idk if we should be importing cross libraries

Reply:
It's fine to import here, as it's a coupled test library.
You should factor the flag into a separate TS file, so it can be included in this file's compilation.

Reply:
You should remove these comments now that you've exported the enum.